package.json

```diff
@@ -3,18 +3,30 @@
   "version": "2.0.0",
   "main": "./src/customfunctions.js",
   "scripts": {
-    "start": "start npm run dev-server && start npm run watch && npm run sideload",
+    "start": "start npm run packager && start npm run dev-server && start npm run watch && npm run sideload",
     "start-web": "start npm run dev-server && start npm run watch",
     "build": "webpack -p --mode production",
-    "dev-server": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 8081 --hotOnly",
+    "dev-server": "webpack-dev-server --mode development --https --key ./certs/server.key --cert ./certs/server.crt --cacert ./certs/ca.crt --port 8080 --hotOnly",
+    "enable-debugging": "office-addin-dev-settings enable-debugging manifest.xml && office-addin-dev-settings set-source-bundle-url manifest.xml --path index.win32",
+    "packager": "haul start --config ./packager/haul.config.js",
     "sideload": "office-toolbox sideload -m manifest.xml -a excel",
     "watch": "webpack -p --mode development --watch"
   },
   "devDependencies": {
+    "@types/es6-collections": "^0.5.29",
```

Comment:
Need to make the packager separate so we don't have the react-native references.